.appveyor.yml

```diff
@@ -30,13 +29,13 @@ install:
   - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
   - cmd: conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
   - cmd: conda update conda
-  - cmd: conda config --remove channels defaults --force
   - cmd: conda config --add channels conda-forge --force
+  - cmd: conda config --set channel_priority strict
```

Comment:
@czender the important change was this one b/c latest `conda` can use the `strict` mode and make `defaults` and `conda-forge` channel work nice together, so there is no need for `conda-forge` to vendorize packages from `defaults` anymore.

Reply:
Yes, thanks much @ocefpaf. This gets us past the prior impasse to a point where now it's breaking to a problem that I know how to solve :)

Reply:
Great! Feel free to ping me anytime you have problems like that.